src/Pretzel/Program.cs

```diff
@@ -17,27 +17,35 @@ namespace Pretzel
     internal class Program
     {
         [Import]
-        private CommandCollection Commands { get; set; }
+        public CommandCollection Commands { get; set; }
+
+        [Export]
+        public IFileSystem FileSystem { get; set; } = new FileSystem();
+
+        [Export]
+        public BaseParameters Parameters { get; set; }
```

Comment:
I don't think that BaseParameters needs to be exported, or am I missing something?

Reply:
No they don't need to be exported